containers/daap-query-data/src/var/task/query_data.py

```diff
@@ -0,0 +1,124 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+from data_platform_api_responses import format_response_json, response_status_404
+from data_platform_logging import DataPlatformLogger
+from http import HTTPStatus
+import os
+import time
+
+
+athena_client = boto3.client('athena')
+
+def start_query_execution_and_wait(
+        athena_client,
+        database_name: str,
+        sql: str,
+        logger:DataPlatformLogger
+    ) -> str:
+        """
+        runs query for given sql and waits for completion
+        """
+        try:
+            res = athena_client.start_query_execution(
+                QueryString=sql,
+                QueryExecutionContext={"Database": database_name},
+                WorkGroup="data_product_workgroup",
```

Comment:
I wonder if we should also store this constant in the paths library. Not sure if it's gonna change at some point.

Reply:
WorkGroup="data_product_workgroup" is coded in few of our lambda. i will add a ticket to make as constant in seperate ticket